src/Bindings/PluginLua.h

```diff
@@ -270,6 +222,28 @@ class cPluginLua :
 
 	/** Releases all Lua references, notifies and removes all m_Resettables[] and closes the m_LuaState. */
 	void Close(void);
+
+	/** Removes all WebTabs currently registered for this plugin from the WebAdmin. */
+	void ClearWebTabs(void);
+
+	/** Calls a hook that has the simple format - single bool return value specifying whether the chain should continue. */
```

Comment:
Is it perhaps usefull to note here how the other hooks are called? Like, do they have a specific method here, or is that done manually somewhere else?